CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
-2.2.0 (Next)
+2.2.0 (04/13/2016)
 ============
 
-* Your contribution here.
```

Comment:
So do you want to have something like this?

2.2.0 (Next)
============

* Your contribution here.
* [#23] Supports Windows - @jonyeezs

Sorry this my first time i'm contributing to another project. So quite
fuzzy on what you're expecting here.


On Thu, Apr 14, 2016 at 12:38 AM, Daniel Doubrovkine (dB.) @dblockdotorg <
notifications@github.com> wrote:

> In CHANGELOG.md
> <https://github.com/dblock/guard-rack/pull/23#discussion_r59559545>:
>
> >  ============
> >
> > -* Your contribution here.
>
> Put this back please.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/dblock/guard-rack/pull/23/files/cbd86e32e4d5f74de0e4a39dc92f539e4eba95b3#r59559545>
>



-- 
"A letter without content is meaningless,
 a heart without honesty is cold;
So a letter with you in mind and
a heart that beats with sincerity,
I send this to you"
- Jonathan Yee